_data/leads.yml

```
@@ -1,39 +1,50 @@
+  image: James.jpg
+  bio: Kylie McKeon is a junior and the business lead of the South Eugene Robotics Team. She joined the team freshman year and loved the business team immediately. Kylie loves helping the team raise money and spreading the word about robotics. In her free time outside robotics, Kylie enjoys reading, practicing circus, and helping out clubs around South Eugene High School.
 
-- name: Sander Moffit
-  subteam: Strategy
-  image: sander.jpg
-  bio: Sander Moffitt is a junior and strategy co-lead for South Eugene Robotics Team. This is her third year on SERT. She fell in love with robotics at the age of eight after building a LEGO robot named “BALL-E” that shot foam balls at her cat whenever it went near her bedroom. She has since evolved to leading robot design discussions and running match strategy. Outside of robotics, Sander enjoys editing her novel, painting, and coming up with creative ways to torment her little sister.
+- name: Keegan Gilds
+  subteam: Co-Strategy
+  image: Sander.jpg
+  bio: Keegan is a senior and co-strategy lead with William on the South Eugene Robotics Team. During robotics he can often be found cleaning, doing menial tasks mediocrely, or scouting in the stands at competitions. Outside of robotics he’s probably grinding piano, doing chemistry homework, or dreaming about doing chemistry homework (literally). He’s also always ready to see a play.
 
 - name: William Hou
-  subteam: Strategy
-  image: william.jpg
-  bio: William is a junior and co-strategy lead along with Sander on the South Eugene Robotics Team. This is William’s third year on the team, and he is also a part of the software subteam. He loves watching robots at competitions and helping organize scouting so that the team has useful data before each match. Outside of school and robotics, William enjoys biking, talking with his friends, and playing online games.
+  subteam: Co-Strategy
+  image: William.jpg
+  bio: William is a senior and co-strategy lead with Keegan on the South Eugene Robotics Team. He loves watching robots at competitions and analyzing data collected at competition to help the team gain a competitive advantage and play at our best. When he’s not strategizing, William also likes to dabble with robot code, edit videos, and help out with outreach projects. Outside of school and robotics, William enjoys biking, hanging out with his friends, playing online games, and trying not to burn down the kitchen while he’s trying out new recipes. 
+
+- name: Ben Weber
+  subteam: Co-Safety
+  image: Ben.jpg
+  bio: Ben Weber is a senior and a co-lead of the safety team. Ben has been involved with FIRST since FIRST Lego League. On SERT, Ben only cares about one thing, and one thing only: the safety for everyone around. Outside of robotics, Ben is on that grind playing water polo and French Horn. In his free time, Ben is busy reading, watching Netflix, and chilling with friends.
```

This will need quotes since it has a colon.

```suggestion
  bio: "Ben Weber is a senior and a co-lead of the safety team. Ben has been involved with FIRST since FIRST Lego League. On SERT, Ben only cares about one thing, and one thing only: the safety for everyone around. Outside of robotics, Ben is on that grind playing water polo and French Horn. In his free time, Ben is busy reading, watching Netflix, and chilling with friends."
```

---

Just, wondering, should the .jpgs have quotes or just the text?

---

They shouldn't need it. Only strings containing some special characters (e.g `:`) need quotes. In this case Ben's bio has a colon so I think it'll need quotes.

---

thanks for the help!